### ISSUE TITLE
Units on Packet Retrans Wrong

### ISSUE THREAD

**Issue by collina** [opentechinstitute/mlab-observatory#53]
Input is in rate, not percentage, but Observatory seems to not make the change before displaying it as percentage?

**Comment by mtlynch** [comment]
Good catch. This made me realize Telescope doesn't document its output format, so I filed https://github.com/m-lab-tools/telescope/issues/39

Observatory just needs to multiply by 100, correct? e.g. 0.015 = 1.5%
